edge_test/fs_test.go

```diff
@@ -0,0 +1,138 @@
+package edge_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/minio/kes/edge"
+)
+
```

Comment:
Maybe it makes sense to use CLI flags to customize the tests.
```
var FSPath string

func init() {
	flag.StringVar(&FSPath, "fs.path", "", "Path used for FS tests")
}
```

Then you can run: 
```
go test -v -fs.path=<some-path>
```